internal/page/fixtures/donor.go

```diff
@@ -209,9 +211,16 @@ func updateLPAProgress(
 	voucherStore *voucher.Store,
 ) (*donordata.Provided, []func(context.Context, *lpastore.Client, *lpadata.Lpa) error, error) {
 	var fns []func(context.Context, *lpastore.Client, *lpadata.Lpa) error
-
 	if data.Progress >= slices.Index(progressValues, "provideYourDetails") {
-		donorDetails.Donor = makeDonor(data.DonorEmail)
+		if data.DonorFirstNames == "" {
+			data.DonorFirstNames = "Sam"
+		}
+
+		if data.DonorLastName == "" {
+			data.DonorLastName = "Smith"
+		}
```

Comment:
To make e2e tests easier without adding to the huge pile of Sam Smith cases in Sirius integration.